src/components/Birthday/Birthday.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { BsCalendar3 } from 'react-icons/bs'
```

Comment:
This is imported but do you use it?

Reply:
We will use this in another file so we've just deleted this line.